spec/services/hyrax/file_set_characterizer_spec.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.describe Hyrax::FileSetCharacterizer do
+  subject(:characterizer) { described_class.new(source: file_set) }
```

Comment:
I have a slight trivial preference for adding a newline between subjects and lets but `/shrug`